postcss.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = () => ( {
+	plugins: {
+		'postcss-custom-properties': {
+			importFrom: [ require.resolve( '@automattic/calypso-color-schemes' ) ],
+			preserve: true,
```

Comment:
`true` is already the default for `preserve`, can be left out.

Reply:
5782caba35